### DISCUSSION TITLE
Heads up: lots of closed issues getting assigned

### DISCUSSION THREAD

**Discussion by coryan** [googleapis/google-cloud-cpp#5788] [General]
Hi,

One of my colleagues (hi @JustinBeckwith) is normalizing closed issues for easier analysis.  Just in case you get a weird notification from the past.

**Comment by ghost** [comment]
I don't know if this impacts the analysis, but a lot of the PR's [here](https://github.com/googleapis/google-cloud-cpp/milestone/22?closed=1) were implemented by @vnvo2409 by assigned to Carlos.
